bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/templates/TemplateListSelectionPage.java

```diff
@@ -80,15 +129,20 @@ private void selectInitialTemplate() throws IOException
 	{
 		templateViewer.reveal(getInitialTemplateId());
 		templateViewer.setSelection(new StructuredSelection(getInitialTemplateId()), true);
-
 		String description = new TemplatesManager().getDescription(getInitialTemplateId());
 		setDescriptionText(description);
 	}
 
 	@Override
-	public boolean isPageComplete()
+	public void selectionChanged(SelectionChangedEvent event)
 	{
-		return true; // will always to finish the page without template selection also
```

Comment:
If this is removed, are we allowing to create a default template project even without a selection?

Reply:
I forgot to add it after changes, thank you for the point. Returned it back.